### PR TITLE
User can click on business to view details

### PR DESCRIPTION
**Pull request by rrundle**

### CODE REVIEW

File: search.html [hunk 1]
```diff
@@ -8,20 +8,22 @@
     <script src="https://use.fontawesome.com/7fdda3cfec.js"></script>
   </head>
   <body>
-    <div class="logo">
-      <img src="http://i.imgur.com/M6AJfjq.jpg">
-    </div>
+    <a href="javascript:history.go(0);">
```
Comment (ronperris) [inline search.html]: Don't inline JavaScript. Remove this.

File: search.js [hunk 2]
```diff
@@ -208,13 +227,206 @@ function renderItem(item) {
...
   review.textContent = item.review;
   biz.appendChild(review);
 
   return biz;
 }
+
+//user can click on business from results to view it
+var content = document.querySelector(".content");
+
+function loadBusiness (businesses, id) {
```
Comment (ronperris) [inline search.js]: Remove space after function name.

File: search.js [hunk 3]
```diff
@@ -208,13 +227,206 @@ function renderItem(item) {
...
+  empty(content)
+  for (i = 0; i < businesses.length; i++) {
+    if (businesses[i].id == id) {
+      var results = renderBusiness(businesses[i]);
+      content.appendChild(results);
+    }
+  }
+}
+
+//event listener for click on any businesses
```
Comment (ronperris) [inline search.js:256]: This should probably be one event listener, with a switch statement inside it.

**Review by ronperris — CHANGES_REQUESTED** [review]
Fix the commented areas.

File: search.html (hunk 1 above)
Comment (rrundle) [reply search.html]: Done.

File: search.js (hunk 2 above)
Comment (rrundle) [reply search.js]: Done.

File: search.js (hunk 3 above)
Comment (rrundle) [reply search.js:256]: Done. One event listener with conditionals.

**Pull request merged**